CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # CHANGELOG
 
+- Remove `areas` field from BusinessSupportEdition
```

Comment:
That's the way I've always been told to do it, so that's the way I've always done it. I'll remember that for future reference though.